doc/topics/releases/3005.rst

```diff
@@ -5,3 +5,122 @@ Salt 3005 Release Notes - Codename Phosphorus
 =============================================
 
 Salt 3005 is currently under development.
+
+New Packages Available
+----------------------
+
+With the release of Salt 3005 we are pleased to announce the
+new packages using tiamat and pyinstaller are now out of beta
+and ready for production. These new packages are self-contained
+binaries of Salt, that include Python and the default requirements.
+This allows users to use Salt without Python being installed on
+the box.
+
+Any new OS platforms we support going forward will only include tiamat
+packages. For this release, this includes Centos 8 Stream, Centos 9 Stream
+and Ubuntu 22.04.
+
+On the day of the Phosphorus release the instructions on how to install the
+new tiamat packages will be available on https://repo.saltproject.io for each
+platform. If you want to test out the packages today you can install them
+from https://repo.saltproject.io/salt-dev/py3/ using the correct directry
```

Comment:
`directly`?

Reply:
It should be directory. I have fixed this.